src/navigation/reroute.js

```diff
@@ -41,23 +39,29 @@ export function reroute(pendingPromises = [], eventArguments) {
     });
   }
 
-  const appsThatChanged = [];
+  const [
+    appsToUnload,
+    appsToUnmount,
+    appsToLoad,
+    appsToMount,
+  ] = getAppChanges();
+  let appsThatChanged;
 
   if (isStarted()) {
     appChangeUnderway = true;
+    appsThatChanged = appsToUnload
+      .concat(appsToLoad)
+      .concat(appsToUnmount)
+      .concat(appsToMount);
```

Comment:
I think you can just call concat once and comma separate them

Reply:
👍 updated